metaseq/trainer.py

```diff
@@ -809,6 +809,10 @@ def maybe_no_sync():
             )
             grad_norm = torch.tensor(0.0).cuda()
             self.zero_grad()
+        except SpikeError as e:
+            overflow = True
+            logger.info(str(e))
+            self.zero_grad()
```

Comment:
This matches how overflow skips and gradient skips are handled

Reply:
I hope this works with skipping batches from pre-emption but i guess it should.